[new product] icinga web (#4437)
captn3m0/endoflife.date: products/icingaweb2.md

---

> title: Icinga Web 2

Reviewer: ```suggestion
title: Icinga Web
```

category: server-app
tags: php-runtime
iconSlug: icinga

> permalink: /icingaweb2

Reviewer: ```suggestion
permalink: /icinga-web
alternate_urls:
- /icingaweb
```

-   releaseCycle: "2.7"
    releaseDate: 2019-07-30
    support: 2020-06-08
    eol: 2021-07-12
    latest: "2.7.6"
    latestReleaseDate: 2021-07-27

-   releaseCycle: "2.6"
    releaseDate: 2018-07-19
    support: 2019-07-30
    eol: 2020-06-08
    latest: "2.6.4"
    latestReleaseDate: 2020-08-17

-   releaseCycle: "2.5"
    releaseDate: 2017-11-27
    support: 2018-07-19
    eol: true
    latest: "2.5.3"
    latestReleaseDate: 2018-04-27

-   releaseCycle: "2.4"
    releaseDate: 2016-12-13
    support: 2017-11-27
    eol: true
    latest: "2.4.2"
    latestReleaseDate: 2017-09-28

-   releaseCycle: "2.3"
    releaseDate: 2016-04-13
    support: 2016-12-13
    eol: true
    latest: "2.3.4"
    latestReleaseDate: 2016-06-23

-   releaseCycle: "2.2"
    releaseDate: 2016-02-29
    support: 2016-04-13
    eol: true
    latest: "2.2.2"
    latestReleaseDate: 2016-06-09

-   releaseCycle: "2.1"
    releaseDate: 2015-11-16
    support: 2016-02-29
    eol: true
    latest: "2.1.4"
    latestReleaseDate: 2016-06-09

-   releaseCycle: "2.0"
    releaseDate: 2015-10-02
    support: 2015-11-16
    eol: true
    latest: "2.0.0"
    latestReleaseDate: 2015-10-02

> ---

Reviewer: Add v1 in the same page.

```suggestion

-   releaseCycle: "1"
    releaseDate: 2010-06-30
    support: 2018-12-31
    eol: 2018-12-31
    latest: "1.14.1"
    latestReleaseDate: 2017-12-19
---
```

> > [Icinga Web 2](https://icinga.com/docs/icinga-web/latest/doc/01-About/) is a web-based graphical user interface for the Icinga2 monitoring system. 

Reviewer: ```suggestion
> [Icinga Web](https://icinga.com/docs/icinga-web/latest/doc/01-About/) is a web-based graphical user interface for the Icinga2 monitoring system.
```

> Since 2.6, Icinga Web 2 follows an N-1 support strategy  
> (meanings that the 2 most recent major versions receive security and bug fixes)  
> The N-2 release may [in certain cases](https://github.com/Icinga/icingaweb2/blob/main/SECURITY.md
> ) receive critical security patches

Reviewer: slightly less confusing:

```suggestion
The 2 most recent releases receive security and bug fixes. 
The N-3 release may receive critical security patches
[in certain cases](https://github.com/Icinga/icingaweb2/blob/main/SECURITY.md)
but this is not guaranteed.
```

    support: 2019-07-30
    eol: 2020-06-08
    latest: "2.6.4"
    latestReleaseDate: 2020-08-17

-   releaseCycle: "2.5"
    releaseDate: 2017-11-27
    support: 2018-07-19
    eol: true
    latest: "2.5.3"
    latestReleaseDate: 2018-04-27

-   releaseCycle: "2.4"
    releaseDate: 2016-12-13
    support: 2017-11-27
    eol: true
    latest: "2.4.2"
    latestReleaseDate: 2017-09-28

-   releaseCycle: "2.3"
    releaseDate: 2016-04-13
    support: 2016-12-13
    eol: true
    latest: "2.3.4"
    latestReleaseDate: 2016-06-23

-   releaseCycle: "2.2"
    releaseDate: 2016-02-29
    support: 2016-04-13
    eol: true
    latest: "2.2.2"
    latestReleaseDate: 2016-06-09

-   releaseCycle: "2.1"
    releaseDate: 2015-11-16
    support: 2016-02-29
    eol: true
    latest: "2.1.4"
    latestReleaseDate: 2016-06-09

-   releaseCycle: "2.0"
    releaseDate: 2015-10-02
    support: 2015-11-16
    eol: true
    latest: "2.0.0"
    latestReleaseDate: 2015-10-02
---

> [Icinga Web 2](https://icinga.com/docs/icinga-web/latest/doc/01-About/) is a web-based graphical user interface for the Icinga2 monitoring system. 

Since 2.6, Icinga Web 2 follows an N-1 support strategy  

> (meanings that the 2 most recent major versions receive security and bug fixes)  

Author: > Since 2.6, Icinga Web 2 follows an N-1 support strategy  

before 2.6, release are in EOL = true

I base this information on the release date of major versions and their latest available versions.
From v2.6, we see that version N-1 is still supported even after the release of version N
